dashboard/app/models/level_group.rb

```diff
@@ -87,4 +88,112 @@ def pages
   def plc_evaluation?
     levels.map(&:class).uniq == [EvaluationMulti]
   end
+
+  # Surveys: How many students must complete a survey before any results are shown.
+  SURVEY_REQUIRED_SUBMISSION_COUNT = 5
+
+  # Surveys: Given a sublevel, and the known response to it, return a result object.
+  def self.get_sublevel_result(sublevel, sublevel_response)
+    sublevel_result = {}
+    if sublevel_response
+      case sublevel
+      when TextMatch, FreeResponse
+        student_result = sublevel_response["result"]
+        sublevel_result[:result] = student_result
```

Comment:
Any reason not to combine these lines to `sublevel_result[:result] = sublevel_response["result"]`?